src/pages/components/footer.mdx

```diff
@@ -24,176 +26,215 @@ import ResourceLinks from 'components/ResourceLinks';
 
 ## Overview
 
-Default footer includes all high-level corporate navigation, the required legal links, the optional adjunct legal links, the locale selector and branding. This footer is generated automatically, based on the country and language code that you set on your page in the metadata.
+The Footer is a required component on all IBM.com pages. There are three sizes available: [Default](#default), [Short](#short) and [Micro](#micro), and each size has a specific use case. The Footer includes the IBM logo, navigational links, Locale or Language selector and required legal links.
```

Comment:
```suggestion
The Footer is a required component on all IBM.com pages. There are three sizes available: [default](#default), [short](#short) and [micro](#micro), and each size has a specific use case. The Footer includes the IBM logo, navigational links, locale or language selector and required legal links.
```

Suggest using exact prop/attribute cases to avoid confusion when they're used.